playbooks/openshift-monitor-availability/README.md

```diff
@@ -0,0 +1,26 @@
+# OpenShift Availability Monitoring
+
```

Comment:
Note that this is unsupported for OCP at this time.

Reply:
Done

Reply:
FYI this note (and the meat of the docs) are in the role README.